dev/AppNotifications/ShellLocalization.cpp

```diff
@@ -167,7 +167,8 @@ void WriteHIconToPngFile(wil::unique_hicon const& hIcon, _In_ PCWSTR pszFileName
     THROW_IF_FAILED(spStreamOut->Commit(STGC_DANGEROUSLYCOMMITMERELYTODISKCACHE));
 }
 
-bool IsIconFileExtensionSupported(std::filesystem::path const& iconFilePath)
+
+bool Microsoft::Windows::AppNotifications::ShellLocalization::IsIconFileExtensionSupported(std::filesystem::path const& iconFilePath)
```

Comment:
`const std::filesystem::path&` ?

Reply:
https://hackingcpp.com/cpp/design/east_vs_west_const.html

Reply:
Yep good to know.